### PR TITLE
Update AttributeScore to Allow Editing

### LINKED ISSUE #62: Update AttributeScore to Allow Editing

### PR DESCRIPTION
**Pull request by shalsten**
Closes #62

### CODE REVIEW

File: src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor [hunk 1]
```diff
@@ -1,14 +1,17 @@
 ﻿@using DMPowerTools.Core.Models;
 
-<MudItem xs="1">
-    <MudText Typo="Typo.h6" Color="ModifierColor">@Modifier</MudText>
+<MudItem xs="3" sm="2" md="1">
+    <MudText Typo="Typo.h5" Color="ModifierColor">@Modifier</MudText>
```
Comment (benjaminsampica) [inline src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor:4]: Does the modifier actually update okay when the RawPoints change?

File: src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-    <MudText Typo="Typo.subtitle1">@RawPoints</MudText>
+    <span>
+        <MudNumericField @bind-Value=@RawPoints ReadOnly=@ReadOnly Class="mt-0" />
+    </span>
 </MudItem>
 
 @code {
     [Parameter, EditorRequired] public int RawPoints { get; set; }
     [Parameter, EditorRequired] public string Name { get; set; }
+    [Parameter] public bool ReadOnly { get; set; } = true;
```
Comment (benjaminsampica) [inline src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor:14]: Did you want to hook this up to the Import as `False` and make sure it updates okay? Or were you thinking another card?

File: src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor (hunk 1 above)
Comment (shalsten) [reply src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor:4]: It does, it updates on each arrow click on the MudNumericField, if applicable, or when focus is lost when entering numbers directly into the text portion.

File: src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor (hunk 2 above)
Comment (shalsten) [reply src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor:14]: I was thinking that would be done with #20. Might need one more story too to allow editing after a creature has been added.

**Review by benjaminsampica — APPROVED** [review]

File: src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor (hunk 1 above)
Comment (benjaminsampica) [reply src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor:4]: neat!

**Pull request merged**